test/external_project/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+
+cmake_minimum_required (VERSION 3.4)
+project (seqan3_test_external_project CXX)
+
+include (../seqan3-test.cmake) # for SEQAN3_EXTERNAL_PROJECT_CMAKE_ARGS, SEQAN3_VERSION
+include (ExternalProject)
+
+set (SEQAN3_ROOT "${CMAKE_CURRENT_LIST_DIR}/../../")
+
+if (NOT ${CMAKE_VERSION} VERSION_LESS 3.14) # cmake 3.14 version is needed to install seqan3.
+    include (install-seqan3.cmake)
+endif ()
+
+option (SEQAN3_EXTERNAL_PROJECT_FIND_DEBUG_MODE
+        "Enable this option if you want to get a detailed list which paths were considered for find_package(...)"
+        false)
+
+if (NOT ${CMAKE_VERSION} VERSION_LESS 3.14) # cmake 3.14 version is needed to install cmake 3.4.
+    set (SEQAN3_EXTERNAL_PROJECT_CMAKE_COMMAND "${CMAKE_COMMAND}")
+    include (use-cmake3.4.cmake) # ensure that seqan3 can be used with cmake 3.4
+else ()
+    set (SEQAN3_EXTERNAL_PROJECT_CMAKE_COMMAND "${CMAKE_COMMAND}")
+endif ()
+
```

Comment:
```suggestion

# Since the above is a bit confusing, here is a list what happens:
#
# If your current cmake version is < 3.14 we
#  * test test/external_project/seqan3_submodule_add_subdirectory/CMakeLists.txt with your current cmake version
#  * test test/external_project/seqan3_submodule_find_package/CMakeLists.txt with your current cmake version
#  * DON'T test test/external_project/seqan3_installed/CMakeLists.txt, because we need 3.14 to be able to
#    package seqan3 to execute the test.
#  * DON'T test test/external_project/seqan3_fetch_content_zip/CMakeLists.txt, because we need 3.14 in order to use
#    fetch_content
#
# If your current cmake version is >= 3.14 we
#  * we download cmake 3.4
#  * test test/external_project/seqan3_submodule_add_subdirectory/CMakeLists.txt with cmake 3.4
#  * test test/external_project/seqan3_submodule_find_package/CMakeLists.txt with cmake 3.4
#  * test test/external_project/seqan3_installed/CMakeLists.txt with cmake 3.4
#  * test test/external_project/seqan3_fetch_content_zip/CMakeLists.txt with your current cmake version

```

Reply:
@eseiler Please edit it to your liking and then commit :)